src/main/java/com/google/api/gax/core/Page.java

```diff
@@ -37,50 +39,26 @@
  * object also provides methods to retrieve additional pages using the page token, and to get the
  * API request and response objects.
  */
-public interface Page<RequestT, ResponseT, ResourceT> extends Iterable<ResourceT> {
+public interface Page<ResourceT> extends Iterable<ResourceT> {
```

Comment:
@lesv We are planning to remove the `Page` interface in google-cloud-java and use the `Page` defined in GAX. One difference between the two is that `Page` in g-c-j has a `getValues()` method, while `Page` in GAX is an `Iterable`. Do you have a preference as to which approach we use? (See https://github.com/GoogleCloudPlatform/google-cloud-java/pull/1821 for a WIP change to remove the `Page` interface). cc @garrettjonesgoogle